plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/RedshiftQueryRunner.java

```diff
@@ -56,7 +56,12 @@ private RedshiftQueryRunner() {}
     private static final Logger log = Logger.get(RedshiftQueryRunner.class);
 
     private static final String S3_TPCH_TABLES_ROOT = requiredNonEmptySystemProperty("test.redshift.s3.tpch.tables.root");
-    private static final String IAM_ROLE = requiredNonEmptySystemProperty("test.redshift.iam.role");
+    public static final String IAM_ROLE = requiredNonEmptySystemProperty("test.redshift.iam.role");
```

Comment:
We probably make testing on the Redshift a bit more difficult if we demand all this setup (even though it is <optional> for the basic functionality of Redshift.